install/tarsframework/templates/tars.php.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: k8s.tars.io/v1beta1
+kind: TTemplate
+metadata:
+  name: tars.php
+  namespace: tars-dev
+  annotations:
+    "helm.sh/hook": pre-install
+    "helm.sh/hook-weight": "-9"
+spec:
+  content: |
+    <tars>
+      <application>
+        enableset=${enableset}
+        setdivision=${setdivision}
+        <client>
+          locator=${locator}
+          sync-invoke-timeout=20000
+          async-invoke-timeout=20000
+          refresh-endpoint-interval=60000
+          stat=tars.tarsstat.StatObj
+          property=tars.tarsproperty.PropertyObj
+          report-interval=60000
+          modulename=${modulename}
+          sample-rate=100000
+          max-sample-count=50
+        </client>
+        <server>
+          app=${app}
+          server=${server}
+          localip=${localip}
+          local=${local}
+          basepath=${basepath}
+          datapath=${datapath}
+          logpath=${logpath}
+          loglevel=DEBUG
+          logsize=15M
+          log=tars.tarslog.LogObj
+          config=tars.tarsconfig.ConfigObj
+          notify=tars.tarsnotify.NotifyObj
+          logLevel=DEBUG
+          php=/usr/bin/php
+          buffer_output_size=12582912
+          open_tcp_nodelay=1
+          open_eof_check=0
+          open_eof_split=0
+          task_worker_num=1
+          dispatch_mode=2
+          daemonize=1
+        </server>
+      </application>
+    </tars>
+  parent: tars.default
```

Comment:
We should only define the differences from the parent template in child template, 
not copying the content of the parent template to child template.

may be we can update this file content like this:

```yaml
 spec:
  content: |
    <tars>
      <application>
        <server>
          php=/usr/bin/php
          buffer_output_size=12582912
          open_tcp_nodelay=1
          open_eof_check=0
          open_eof_split=0
          task_worker_num=1
          dispatch_mode=2
          daemonize=1
        </server>
      </application>
    </tars>
  parent: tars.default
````